generic_config_updater/generic_updater.py

```diff
@@ -0,0 +1,469 @@
+import json
```

Comment:
How do you integrate the new module with `config` ? Add the test for that integration

Reply:
done

Reply:
I see you modified `config` code, but only integrated with commented code.

Reply:
I might have an earlier commit that had the code commented, but that is un-commented now.